UselessMine.cpp

```diff
@@ -340,13 +340,21 @@ void UselessMine::Event (bz_EventData *eventData)
                             bz_sendTextMessagef(BZ_SERVER, playerID, "You were killed by %s's mine", owner);
                         }
                     }
+                    else
+                    {
+                        //if the owner was killed with their own mine, send a message
+                        bz_sendTextMessagef(BZ_SERVER, BZ_ALLUSERS, "%s was owned by their own mine!", owner);
+                    }
                 }
                 else if (mine.defused)
                 {
-                    dieData->killerID = mine.defuserID;
-
-                    if (playerID != mine.owner)
+                    // Make sure the killer was the server, and the victim was the owner.
+                    if (dieData->killerID == 253 && playerID == mine.owner)
```

Comment:
Wouldn't checking `playerID == mine.owner` ignore any bystanders who happened to be near the defusal SW?

Reply:
oops